git_info/main.py

```diff
@@ -1,32 +1,19 @@
-import click
-from github import Github
+import os
 
+from github import Github
 from stargazers import StargazersExtractor
 from summarizer import GitSummarizer
 
+if __name__ == "__main__":
+    repo_url = os.getenv("repo-url")
```

Comment:
let's use entrypoint - and you can pass arguments in the same way as to python script
```
docker run daria_image -dir bla -url blabla
```

Reply:
and in this case - you don't need to modify this file at all